powershell/VstsTaskSdk/InputFunctions.ps1

```diff
@@ -67,7 +69,7 @@ function Get-Input {
         [switch]$AsInt)
```

Comment:
It's passed when calling Get-SecureInput by splatting the PSBoundParameters automatic variable (using the `@` splat operator).